code/game/objects/structures/plaques/_plaques.dm

```diff
@@ -0,0 +1,207 @@
+		if(!namechoice)
+			return
+		var/descriptionchoice = input(user, "Engrave this plaque's text.", "Plaque Customization")
+		if(!descriptionchoice)
+			return
+		if(!Adjacent(user)) //Make sure user is adjacent still
+			to_chat(user, "<span class='warning'>You need to stand next to the plaque to engrave it!</span>")
+			return
+		user.visible_message("<span class='notice'>[user] begins engraving [src].</span>", \
+							 "<span class='notice'>You begin engraving [src].</span>")
+		if(!do_after(user, 40, target = src)) //This spits out a visible message that somebody is engraving a plaque, then has a delay.
+			return
+		name = "\improper [namechoice]" //We want improper here so examine doesn't get weird if somebody capitalizes the plaque title.
+		desc = "The plaque reads: '[descriptionchoice]'"
+		engraved = TRUE //The plaque now has a name, description, and can't be altered again.
+		user.visible_message("<span class='notice'>[user] engraves [src].</span>", \
+							 "<span class='notice'>You engrave [src].</span>")
+		return
+	if(istype(I, /obj/item/pen))
+		if(engraved)
+			to_chat(user, "<span class='warning'>This plaque has already been engraved, and your pen isn't fancy enough to engrave it anyway! Find a fountain pen.</span>")
+			return
+		to_chat(user, "<span class='warning'>Your pen isn't fancy enough to engrave this! Find a fountain pen.</span>") //Go steal the Curator's.
+		return
+	return ..()
+
+/obj/item/plaque/afterattack(atom/target, mob/user, proximity)
+	. = ..()
+	if(!iswallturf(target) || !proximity)
+		return
+	var/turf/T = target
+	var/turf/userT = get_turf(user)
+	var/obj/structure/plaque/S = new plaque_path(userT) //We place the plaque on the turf the user is standing, and pixel shift it to the target wall, as below.
+	//This is to mimic how signs and other wall objects are usually placed by mappers, and so they're only visible from one side of a wall.
+	var/dir = get_dir(userT, T)
+	switch(dir)
+		if(NORTH)
+			S.pixel_y = 32
+		if(SOUTH)
+			S.pixel_y = -32
+		if(EAST)
+			S.pixel_x = 32
+		if(WEST)
+			S.pixel_x = -32
+		if(NORTHEAST)
+			S.pixel_y = 32
+			S.pixel_x = 32
+		if(NORTHWEST)
+			S.pixel_y = 32
+			S.pixel_x = -32
+		if(SOUTHEAST)
+			S.pixel_y = -32
+			S.pixel_x = 32
+		if(SOUTHWEST)
+			S.pixel_y = -32
+			S.pixel_x = -32
+	user.visible_message("<span class='notice'>[user] fastens [src] to [T].</span>", \
+						 "<span class='notice'>You attach [src] to [T].</span>")
+	playsound(T, 'sound/items/deconstruct.ogg', 50, TRUE)
+	if(engraved)
+		S.name = name
+		S.desc = desc
+		S.engraved = engraved
+	S.obj_integrity = obj_integrity
+	S.setDir(dir)
+	qdel(src)
+
+/obj/item/plaque/Move(atom/new_loc, direct = 0)
+	// Pulling, throwing, or conveying a plaque does not rotate it.
+	var/old_dir = dir
+	. = ..()
+	setDir(old_dir)
+
+/obj/item/plaque/attack_self(mob/user)
+	. = ..()
+	setDir(turn(dir, 90))
```

Comment:
Unless changing direction matters for something, feel free to remove both of these.

Reply:
I was considering that yeah, just wasn't sure. Will do.

Reply:
Done.